### PR TITLE
Feature/df 2004 gradle2mvn

### PR DESCRIPTION
**Pull request by fabiangotzen**

### CODE REVIEW

**Comment by fabiangotzen** [comment]
Yes, I'm aware. 
The dependency can only be found in the jcenter.bintray repository: 

https://jcenter.bintray.com/com/overzealous/remark/

When you use the provided settings.xml, maven should use the libs-releases repository, where i've added jcenter as a virtual repository, so I thought it might be some caching issue, but it doesn't seem to help. @cinhtau  @birom  any ideas? 

It's only for the local build, on jenkins it runs through: https://jenkins.job-room.ch/job/job-ad-service/job/feature%252FDF-2004-gradle2mvn/

**Comment by cinhtau** [comment]
This is ok in my opinion. Our Artifactory retrieves the artifact, though the given settings.xml should worked. The build has no problem, otherwise the checkmarks would be red. https://jenkins.job-room.ch/job/job-ad-service/job/feature%252FDF-2004-gradle2mvn/15/console On my local linux computer it does work too.

File: .mvn/wrapper/MavenWrapperDownloader.java [hunk 1]
```diff
@@ -0,0 +1,117 @@
+/*
```
Comment (seco-mgabor) [inline .mvn/wrapper/MavenWrapperDownloader.java:1]: What is this file used for? 
How does it get compiled?

File: infrastructure/build.gradle [hunk 2]
```diff
@@ -8,8 +8,7 @@ dependencies {
     compile 'ch.admin.seco.alv:alv-logger-starter'
```
Comment (seco-mgabor) [inline infrastructure/build.gradle:8]: Shouldn't be the Gradle related file removed? I assume they are not needed after the migration to maven.

**Comment by seco-mgabor** [comment]
> 
> 
> Yes, I'm aware.
> The dependency can only be found in the jcenter.bintray repository:
> 
> https://jcenter.bintray.com/com/overzealous/remark/
> 
> When you use the provided settings.xml, maven should use the libs-releases repository, where i've added jcenter as a virtual repository, so I thought it might be some caching issue, but it doesn't seem to help. @cinhtau @birom any ideas?
> 
> It's only for the local build, on jenkins it runs through: https://jenkins.job-room.ch/job/job-ad-service/job/feature%252FDF-2004-gradle2mvn/

I couldn't find that repository in the maven wrapper settings file.
The artifact is available at https://mvnrepository.com/artifact/com.overzealous/remark too, if that helps.
I will investigate why this works on Jenkins, locally on my machine it doesn't.
How difficult would it be to remove the dependency to this artifact?

File: infrastructure/build.gradle (hunk 2 above)
Comment (fabiangotzen) [reply infrastructure/build.gradle:8]: Added the mvnw scripts. MavenWrapperDownloader gets called by them. 
Gradle files can be removed in a later step, we discussed that its not yet part of these pull requests at one of our meetings.

**Comment by seco-mgabor** [comment]
I've found a work around for the overzelous dependency:
https://mvnrepository.com/artifact/com.kotcrab.remark/remark
<dependency>
                <groupId>com.kotcrab.remark</groupId>
                <artifactId>remark</artifactId>
                <version>1.2.0</version>
            </dependency>
This has a newer version and can be found in github too:
https://github.com/kotcrab/remark-java/blob/master/src/main/java/com/overzealous/remark/Remark.java

**Comment by fabiangotzen** [comment]
Feel free to push it when it resolves your issue. 
Please comment on a file in the future so I can directly address the issue and resolve it in a github conversation, otherwise the discussion gets a bit convoluted.

**Comment by seco-mgabor** [comment]
Sorry Fabian, I had the intention, but I couldn't find the "comment on file" stuff. I'll look for it better in the future. I'll push my change regarding the dependency then. Thanks!

File: .mvn/wrapper/MavenWrapperDownloader.java (hunk 1 above)
Comment (seco-mgabor) [reply .mvn/wrapper/MavenWrapperDownloader.java:1]: Could you please remove this file? It's no use to keep it here.

**Pull request merged**